composer.json

```
@@ -0,0 +1,11 @@
+{
+    "name": "mmm545/libgamespyquery",
+    "description": "A virion and library to query servers using GS4",
+    "type": "library",
+    "require": {},
```

```suggestion
    "require": {
        "php": ">=8.0"
    },
```
I believe `7.4` also works. But @mmm545 took `8.0` in virion.yml:
https://github.com/mmm545/libgamespyquery/blob/8598c29ef2f3c345006c6c0ad31d01a84c6ed832/virion.yml#L1-L5